cmd/caboose/main.go

```diff
@@ -50,15 +50,17 @@ func main1() int {
 
 			cb, err := caboose.NewCaboose(&caboose.Config{
 				OrchestratorEndpoint: *oe,
-				OrchestratorClient:   http.DefaultClient,
+				OrchestratorClient: &http.Client{
```

Comment:
@lidel @willscott I don't see us creating a default orchestrator HTTP client in the Caboose constructor. Does the IPFS GW always supply an initialised Orchestrator HTTP client ?

Reply:
We pass our own http.Client for thing like a  proper `User-Agent` header:
https://github.com/ipfs/bifrost-gateway/blob/e0b155e26a963acdcbf2665acd218ace929be0fb/blockstore.go#L80-L81